v5_ws_trade.go

```diff
@@ -0,0 +1,189 @@
+package bybit
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"sync"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// V5WebsocketTradeServiceI :
+type V5WebsocketTradeServiceI interface {
+	Start(context.Context, ErrHandler) error
+	Login() error
```

Comment:
I would like to unify.

```suggestion
	Subscribe() error
```

ref. https://github.com/hirokisan/bybit/blob/91aae7b5f568b93a57eeb644cb858981f413f5a3/v5_ws_private.go#L19